### PR TITLE
Remove recipes

### PR DESCRIPTION
**Pull request by dbritto-dev**
## Related Issues or Discussions

Fixes https://github.com/pmndrs/zustand/discussions/1033#discussioncomment-7118251

## Summary

Remove recipes

## Check List

- [x] `yarn run prettier` for formatting code and docs

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -185,11 +185,14 @@ const unsub1 = useDogStore.subscribe(console.log)
 useDogStore.setState({ paw: false })
 // Unsubscribe listeners
 unsub1()
+// Destroying the store (removing all listeners)
+useStore.destroy()
```
Comment (dai-shi) [inline readme.md:189]: Let's remove this as `.destroy()` is deprecated and will be removed in the next major.

File: readme.md [hunk 2]
```diff
@@ -267,6 +270,7 @@ const Component = () => {
     state => (scratchRef.current = state.scratches)
   ), [])
   ...
+}
```
Comment (dai-shi) [inline readme.md:273]: Eliminating it is intentional to make readme as compact as possible.

**Review by dai-shi — COMMENTED** [review]
So, do we not have anything to leave in docs?

File: readme.md (hunk 2 above)
Comment (dbritto-dev) [reply readme.md:271]: Do you want me to remove it? I'd leave it

**Comment by dbritto-dev** [comment]
> So, do we not have anything to leave in docs?

No, we don't. Actually everything on recipes are duplicated on readme, typescript and call outside of react context

File: readme.md (hunk 2 above)
Comment (dai-shi) [reply readme.md:271]: Let's remove it for now as that's the original plan.
```suggestion
```

File: readme.md [hunk 3]
```diff
@@ -187,9 +187,10 @@ useDogStore.setState({ paw: false })
 unsub1()
 
 // You can of course use the hook as you always would
-const Component = () => {
+function Component() {
   const paw = useDogStore((state) => state.paw)
-  ...
+  // ...
+}
```
Comment (dai-shi) [inline readme.md:193]: Let's revert this too.
```suggestion
  ...
```

**Comment by dai-shi** [comment]
https://github.com/pmndrs/zustand/blob/8e8156f1426105e75f4e041c92bb4561e7dd1e61/readme.md?plain=1#L168-L170

We actually want to change this ☝️ for consistency.

**Comment by dbritto-dev** [comment]
> https://github.com/pmndrs/zustand/blob/8e8156f1426105e75f4e041c92bb4561e7dd1e61/readme.md?plain=1#L168-L170
> 
> We actually want to change this ☝️ to just `...` for consistency.

Done

File: readme.md [hunk 4]
```diff
@@ -165,7 +165,7 @@ const useSoundStore = create((set, get) => ({
   sound: 'grunt',
   action: () => {
     const sound = get().sound
-    // ...
+    ...
   },
 }))
```
Comment (dai-shi) [inline readme.md:170]: The point is to reduce lines.
```suggestion
```

**Comment by dbritto-dev** [comment]
@dai-shi all yours

**Review by dai-shi — APPROVED** [review]

**Review by sewera — APPROVED** [review]

**Comment by dbritto-dev** [comment]
@dai-shi you have to merge it

**Pull request merged**